packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -555,6 +555,33 @@ export const updateStackForAPIMigration = async (context: $TSContext, category:
   }
 };
 
+/**
+ * Publish files that Amplify Studio depends on outside the zip file so that can read
+ * without streaming from the zip.
+ */
+const uploadStudioBackendFiles = async (s3: S3, bucketName: string) => {
+  const amplifyDirPath = pathManager.getAmplifyDirPath();
+  const studioBackendDirName = 'studio-backend';
+  // Delete the contents of the studio backend directory first
+  await s3.deleteDirectory(bucketName, studioBackendDirName);
+  // Create a list of file params to upload to the deployment bucket
+  const uploadFileParams = [
+    'cli.json',
+    'amplify-meta.json',
+    'backend-config.json',
+    'schema.graphql',
+    'transform.conf.json',
+    'parameters.json',
+  ]
+    .flatMap(baseName => glob.sync(`**/${baseName}`, { cwd: amplifyDirPath }))
+    .filter(filePath => !filePath.startsWith('backend/'))
```

Comment:
is the idea here to only grab files that are in `#current-cloud-backend`? And if so, why not only look in that directory to begin with?

Reply:
Unfortunately we need the cli.json file as well which is outside `#current-cloud-backend`. The logic is easier this way than special casing that one file.